etc/config/solidity.amazon.properties

```diff
@@ -0,0 +1,175 @@
+compilers=&solc
+defaultCompiler=solc0813
+
+group.solc.compilers=solc036:solc040:solc041:solc042:solc043:solc044:solc045:solc046:solc047:solc048:solc049:solc0410:solc0411:solc0412:solc0413:solc0414:solc0415:solc0416:solc0417:solc0418:solc0419:solc0420:solc0421:solc0422:solc0423:solc0424:solc0425:solc0426:solc050:solc051:solc052:solc053:solc054:solc055:solc056:solc057:solc058:solc059:solc0510:solc0511:solc0512:solc0513:solc0514:solc0515:solc0516:solc0517:solc060:solc061:solc062:solc063:solc064:solc065:solc066:solc067:solc068:solc069:solc0610:solc0611:solc0612:solc070:solc071:solc072:solc073:solc074:solc075:solc076:solc080:solc081:solc082:solc083:solc084:solc085:solc086:solc087:solc088:solc089:solc0810:solc0811:solc0812:solc0813
```

Comment:
Can you add all 3
```
compilerType=solidity
supportsBinary=false
instructionSet=evm
```
for this group in this file?

Reply:
done